WaarpCommon/src/main/java/org/waarp/common/utility/SystemPropertyUtil.java

```diff
@@ -94,6 +109,41 @@ public static void refresh() {
           PROPS.clear();
           PROPS.putAll(newProps);
         }
+        SysErrLogger.FAKE_LOGGER.syserr(IO_NETTY_ALLOCATOR_TYPE + ":" +
+                                        io.netty.util.internal.SystemPropertyUtil
+                                            .get(IO_NETTY_ALLOCATOR_TYPE));
```

Comment:
This is for debugging. It belongs more to the log file than to the error output...